### ISSUE TITLE
Inverse maps

### ISSUE THREAD

**Issue by tobydriscoll** [complexvariables/conformalmapping#51]
We will want to provide a generalized mechanism for inverse maps. What I propose is to have a boolean flag isInverted in each map object. When that flag is set, a different group of evaluation routines is called that (among other things) invoke inverse mapping functions of the map class. The class can choose to provide them, but they are set to throw errors by default.

**Comment by ehkropf** [comment]
Agreed. The `isInverted` property and default inversion error methods should be implemented in the `conformalmap` class.
